index.js

```diff
@@ -2,58 +2,101 @@ const core = require("@actions/core");
 const fs = require("fs");
 const yaml = require("js-yaml");
 const path = require("path");
-const { componentYamlSchemaV1D0 } = require("./schemas");
+const {
+  componentYamlSchemaV1D0,
+  endpointYamlSchemaV0D1,
+  componentConfigYamlSchemaV1beta1,
+} = require("./schemas");
+
+const SourceConfigFileTypes = {
+  COMPONENT_YAML: "component.yaml",
```

Comment:
Adding this in seperate file will increase the reusability. Ignore if not valid

Reply:
ex:- https://github.com/choreo-templates/ballerina-toml-parser/blob/master/enums.js